tests/recipient_validation/test_phone_number.py

```diff
@@ -561,5 +561,5 @@ def test_tv_number_passes(self, phone_number, expected_valid_number):
     )
     def test_international_does_not_normalise_to_uk_number(self, phone_number, expected_error_code):
         with pytest.raises(InvalidPhoneError) as exc:
-            PhoneNumber(phone_number, allow_international=False)
+            PhoneNumber(phone_number, allow_international=False, allow_landline=True)
```

Comment:
We are missing some unit tests: currently there is no test for when we use `PhoneNumber` class, but `allow_landline=False`. So we are never actually testing the happy path with that flag set to false, and we never test the `_raise_if_service_cannot_send_to_landline_but_tries_to` doing its job and raising an error, either.

Let's add the missing tests?

Reply:
Good spot. I've noticed that we also don't have this test for international numbers. Probably something to rectify at a later date.